docs/framework/configure-apps/toc.md

```diff
@@ -9,3 +9,4 @@
 ### [Mapping Object Identifiers to Cryptography Algorithms](map-object-identifiers-to-cryptography-algorithms.md)
 ## [How to: Create a Publisher Policy](how-to-create-a-publisher-policy.md)
 ## [Configuration file schema](file-schema/index.md)
+## [Windows Forms Configuration Section](winforms/index.md)
```

Comment:
This should be removed -- Windows Forms configuration section is a child of Configuration file schema, and it's already listed in the Configuration file schema topic.